download.d.ts

```diff
@@ -2,11 +2,13 @@ declare module "ngrok/download" {
   export default function downloadNgrok(
     callback: (err?: Error) => void,
     options?: {
-      cafilePath: string;
-      arch: string;
-      cdnUrl: string;
-      cdnPath: string;
-      ignoreCache: boolean;
+      cafilePath?: string;
```

Comment:
The only new addition are the last two lines. The other lines are made optional to reflect reality.